jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/SkaffoldSyncMapTemplate.java

```diff
@@ -6,6 +6,7 @@
 import com.google.cloud.tools.jib.api.LayerEntry;
```

Comment:
This one doesn't have copyright either.

Personally I would avoid coupling `LayerEntry` for `JsonTemplate` classes, as a low-level class for a JSON template and just for that. The template thing is at a very low level in the hierarchy. But I don't feel strongly about this, and this certainly isn't to be address in this PR.

Reply:
It's only used to populate it, it's not actually part of the json template. Merely a convenience.